common/lib/xmodule/xmodule/html_module.py

```diff
@@ -76,6 +76,17 @@ def student_view(self, _context):
         """
         return Fragment(self.get_html())
 
+    def student_view_data(self):
```

Comment:
@bradmerlin Don't forget to add the `context` fix here too.

Reply:
@bradmerlin Reminder about this fix required too ^

Reply:
Addressed by https://github.com/edx/edx-platform/pull/15905/commits/0e68323683d8708f748af43c31369f7a3413aead#diff-04a8c30ca1feab9f927c066c88487949R80